### ISSUE TITLE
[Feature request] direct link between english and swedish sites

### ISSUE THREAD

**Issue by vlle1** [datasektionen/bawang-content#358]
Hey @whoever reads this :)

tl;dr: clicking the "english" or "svenska"-button should redirect to the same page, i.e. clicking "english" on an article should lead to the same article in english (if it exists). Note that this is a feature-request, not a bug or complaint :)

first of all, it is really nice that a lot of content on the website is available in english.
However I still usually browse the swedish version (it is good to learn the swedish terms, and sometimes it at least feels more reliable to look at the swedish version) - and if I don't understand something I then intuitively click on the "English" button, before realizing that I have to find the same page again. This is at least at the beginning a bit confusing, as the top "navigation bar" looks different in english and swedish (which is not bad in itself, because the targeted audience is slightly different). 
I can make changes myself as pull request, however the question I would ask people who have previously worked on the webpage is:
- is it possible to change the "switch language" behavior without substantial changes to the architecture?

Other suggestions that improve the website (could be opened as different issues if one feels the need for that):
- show a navigation path at the top of the page (to the right of "Tillbaka" or something)
- prefix translated (english) pages titles with [EN] automatically, at least in the search bar. This avoids confusion while using the search (see picture).

![grafik](https://user-images.githubusercontent.com/36682087/197191728-8b072840-6850-4050-85e2-8d61c43f0faa.png)

**Comment by Herkarl** [comment]
I definitely agree that this would be very nice to have.
Changing this would involve making some changes in a few different repos.
A very quick rundown of the 3 parts that make up our website (a somewhat more complete version is available in the READMEs of the repos):
- [bawang-content](https://github.com/datasektionen/bawang-content): contains the raw content of the website in markdown
- [taitan](https://github.com/datasektionen/taitan): A service that converts the markdown in `bawang-content` to html 
- [bawang](https://github.com/datasektionen/bawang): Defines the structure of the website and generates it based on that

I made an issue about the same thing some time ago in bawang ([bawang#32](https://github.com/datasektionen/bawang/issues/32)), where I think most of the changes should be made.

TL;DR: I think there are two approaches we could do for making this work:
- Each folder in bawang contains a `meta.toml` file that cointains some meta information for the page, we could probably add the path to the enlish page there and work with that
- Restructure `bawang-content` completlyt

**Comment by elmaxe** [comment]
The logic of the switching-language-button resides in the root of the app: https://github.com/datasektionen/bawang/blob/7deedbb359fe75a596a69614ad8085cdfd4955d1/src/components/App.js#L17

**Comment by mathiasmagnusson** [comment]
Would it be reasonable to make the Swedish and English pages have the same urls (except the `en/`)? That would also make it slightly easier to find the file for the English version of a Swedish page when editing this repository.

**Comment by Herkarl** [comment]
I think it would make a lot of sense from a developer perspective, (and what we would have to do for what I had in mind for my second point, i.e having something like a `sv` and a `en` subdir (and other languages if we wanted to)). But I think having localized url:s is a point for usability, and is fairly common. But tbh, probably better to have a nice repo structure.

**Comment by vlle1** [comment]
I argue we should use exactly the same urls.

Suggestion: Differentiate between languages by using a query string at the end (`?lang=en` or something like that).
Use same directories for english and swedish pages and just have some different file name for the english ones. (like pagename.en.md)

If one wants to improve usability, one could of course think about translating the paths. But this should not be done by creating redundant folder structures as done currently.

**Comment by Herkarl** [comment]
@vlle1 your legacy has been fulfilled

**Comment by elmaxe** [comment]
Snyggt byggt, fräsig kärra!

**Comment by vlle1** [comment]
Nice @Herkarl !
